crypto/fipsmodule/self_check/fips.c

```diff
@@ -29,6 +29,9 @@ int FIPS_mode(void) {
 int FIPS_mode_set(int on) { return on == FIPS_mode(); }
 
 #if defined(BORINGSSL_FIPS_140_3)
+
+const char *FIPS_module_name(void) { return "AWSLCCrypto"; }
```

Comment:
What's the purpose of this?

Reply:
This was discussed in team's channel. The decision was keeping this API.

Reply:
We can remove this API before merging `current-upstream-merge-20220411` into `main`.

Reply:
Another thought is to keep open with these experiment FIPS code. Removing these code may bring more work in future upstream stream merge because more formal implementation is still being implemented.